extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -15,24 +15,24 @@ limitations under the License.
 -->
```

Comment:
I'm wondering if we should rename the directory and this file name?

Reply:
Not sure if it'll matter; especially if the content will eventually be imported into ampproject.org.